README.md

```diff
@@ -53,10 +53,12 @@ Script has several options than can be passed using `args` array for the `pre-co
   Default port is specified by `$NAILGUN_PORT` environment variable if it exists. If not, `2113` is taken.
   Alternatively one can use `-p` option to override the port.
 
+- `-S` allows you to set the Scala version for `scalafmt`, e.g. `-S2.12` or `-S2.13`.
+
 - `-t` - passes `--test` to `scalafmt`, that implies no mis-formatted file will be changed instead of returning `1` exit code on any.
 
-- `-v $version` - forces to use specific (default=`1.4.0`) `scalafmt` version.
+- `-v $version` - forces to use specific (default=`1.4.0`) `scalafmt` version. **Note**: `1.4.0` is a significantly old version. Make sure to match this with the version you are using, e.g. `2.6.3`.
```

Comment:
Would have been happy to accept the new defaults 😄

Reply:
@smur89 oh, sorry, I misread between the lines on your original review. If I get a second I will send another PR to remove these comments and update the defaults.

Reply:
No worries @Kazark, appreciate the contribution you made!